sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/controller/container/ThroughputResolveLevel.java

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.throughputControl.controller.container;
+
+public enum ThroughputResolveLevel {
```

Comment:
NIT - ThroughputProvisioningModel or ThroughputProvisioningScope?

Reply:
I like the ThroughputProvisioningScope, will change to it in the next iteration.